### PR TITLE
[Unity] Fix importing tvm.contrib.cutlass

### PR DESCRIPTION
**Pull request by vinx13**
Fixed import error introduced in https://github.com/apache/tvm/pull/15001

### CODE REVIEW

File: python/tvm/relax/backend/contrib/cutlass.py [hunk 1]
```diff
@@ -160,7 +162,7 @@ def _check_matmul(context: PatternCheckContext) -> bool:
 
     lhs_shape = lhs.struct_info.shape.values
     rhs_shape = rhs.struct_info.shape.values
-    return tvm.contrib.cutlass.build.is_shape_valid_for_cutlass_matmul(lhs_shape, rhs_shape)
+    return is_shape_valid_for_cutlass_matmul(lhs_shape, rhs_shape)
```
Comment (masahi) [inline python/tvm/relax/backend/contrib/cutlass.py:165]: Sorry I didn't notice that #15001 didn't revert the change in `contrib/cutlass.py` after the dep of `cutlass/build.py` to `contrib/cutlass.py` has been removed.

Why do we need local import here? I thought it is fine for `contrib/cutlass.py` to depend on `cutlass/build.py`.

Comment (vinx13) [reply python/tvm/relax/backend/contrib/cutlass.py:164]: yeah it's fine, I have updated the PR

**Review by masahi — APPROVED** [review]
